### PR TITLE
clean up some objects

### PR DESCRIPTION
**Pull request by randomsalience**
Matched some rels that were already almost done: d_a_obj_bbox, d_a_obj_lv4Gate, d_a_obj_stoneMark, d_a_obj_wsword, d_a_obj_zcloth

### CODE REVIEW

File: rel/d/a/obj/d_a_obj_lv4Gate/d_a_obj_lv4Gate.cpp [hunk 1]
```diff
@@ -1,291 +1,239 @@
...
+        }
+
+        setBaseMtx();
+    }
+
+    return phase;
+}
+
+/* 80C5EE54-80C5EEA4 0003D4 0050+00 1/0 0/0 0/0 .text            Execute__11daLv4Gate_cFPPA3_A4_f */
+int daLv4Gate_c::Execute(f32 (**param_0)[3][4]) {
```
Comment (Trueffeloot) [inline rel/d/a/obj/d_a_obj_lv4Gate/d_a_obj_lv4Gate.cpp:104]: `f32 (**param_0)[3][4]` is actually an `Mtx** param_0`. This hasn't been changed back then but you could optimize it to that now (also in the function declaration)

File: rel/d/a/obj/d_a_obj_wsword/d_a_obj_wsword.cpp [hunk 2]
```diff
@@ -1,326 +1,149 @@
...
+void daObjWSword_c::initBaseMtx() {
+    mpModel->setBaseScale(mScale);
+    setBaseMtx();
+}
+
+/* 80D3B9D4-80D3BA40 0000D4 006C+00 2/2 0/0 0/0 .text            setBaseMtx__13daObjWSword_cFv */
+void daObjWSword_c::setBaseMtx() {
+    mDoMtx_stack_c::transS(current.pos.x, current.pos.y, current.pos.z);
+    mDoMtx_stack_c::ZXYrotM(shape_angle.x, shape_angle.y, shape_angle.z);
+    PSMTXCopy(mDoMtx_stack_c::get(), mpModel->mBaseTransformMtx);
```
Comment (Trueffeloot) [inline rel/d/a/obj/d_a_obj_wsword/d_a_obj_wsword.cpp:45]: I remember that I worked on this TU and I didn't realise earlier that I should have used the `setBaseTRMtx` inline at this spot lol. Could you replace this line with `mpModel->i_setBaseTRMtx(mDoMtx_stack_c::get());` for me?

File: rel/d/a/obj/d_a_obj_zcloth/d_a_obj_zcloth.cpp [hunk 3]
```diff
@@ -1,310 +1,114 @@
...
+#include "JSystem/JKernel/JKRHeap.h"
+#include "f_op/f_op_actor_mng.h"
+#include "d/com/d_com_inf_game.h"
+#include "d/d_item_data.h"
+#include "d/a/d_a_itembase_static.h"
+#include "d/d_procname.h"
+
+/* 80D3EE18-80D3EE54 000078 003C+00 1/1 0/0 0/0 .text            initBaseMtx__13daObjZCloth_cFv */
+void daObjZCloth_c::initBaseMtx() {
+    mpModel->mBaseScale = mScale;
```
Comment (Trueffeloot) [inline rel/d/a/obj/d_a_obj_zcloth/d_a_obj_zcloth.cpp:16]: similiar thing to daObjWSword_c. I worked on this and I didn't use a few inlines. This line should be `mpModel->setBaseScale(mScale);`

File: rel/d/a/obj/d_a_obj_zcloth/d_a_obj_zcloth.cpp [hunk 4]
```diff
@@ -1,310 +1,114 @@
...
+    setBaseMtx();
+}
+
+/* 80D3EE54-80D3EEC0 0000B4 006C+00 2/2 0/0 0/0 .text            setBaseMtx__13daObjZCloth_cFv */
+void daObjZCloth_c::setBaseMtx() {
+    mDoMtx_stack_c::transS(current.pos.x, current.pos.y, current.pos.z);
+    mDoMtx_stack_c::ZXYrotM(shape_angle.x, shape_angle.y, shape_angle.z);
+    MTXCopy(mDoMtx_stack_c::now, mpModel->mBaseTransformMtx);
```
Comment (Trueffeloot) [inline rel/d/a/obj/d_a_obj_zcloth/d_a_obj_zcloth.cpp:24]: Again my bad. `mpModel->i_setBaseTRMtx(mDoMtx_stack_c::get());`

**Review by Trueffeloot — COMMENTED** [review]
Lgtm! Good job fixing these nonmatchings. I commented on a few spots which can be optimized. Those were our or mostly my "mistakes" though which can be optimized in this pr now that these files have been picked up again.

**Comment by randomsalience** [comment]
Okay the issues have been fixed

**Comment by Trueffeloot** [comment]
Thanks! I'll merge

**Review by Trueffeloot — APPROVED** [review]

**Pull request merged**